components/Feature/TopicExplorer/index.js

```diff
@@ -38,21 +52,29 @@ const TopicExplorer = (props) => {
       <h2>Discuss a topic</h2>
       <div className="govuk-form-group govuk-!-margin-bottom-7">
        
-      <div class="govuk-!-padding-bottom-4" id='example-search'>Try searching for keywords like{' '}
+      <div class="govuk-!-padding-bottom-4" id='example-search'>
+        <label for="text-input">Try searching for keywords like{' '}
+        </label>
           <button className='button-as-link govuk-!-padding-0' data-search-term='coronavirus' onClick={populateInput} data-testid='coronavirus'>coronavirus</button>,{' '}
           <button className='button-as-link govuk-!-padding-0' data-search-term='food' onClick={populateInput} data-testid='food'>food</button>,{' '}
           <button className='button-as-link govuk-!-padding-0' data-search-term='health' onClick={populateInput}>health</button>, or{' '}
           <button className='button-as-link govuk-!-padding-0' data-search-term='benefits'onClick={populateInput}>benefits</button>
         </div>
-
         <input
+          id="text-input"
+          list='input-tags'
           type="text"
           className="govuk-input govuk-input--width-20"
           value={searchTerm}
           onChange={onSearchChange}
         />
-
-
+        <datalist id="input-tags">  
+        {
+            tags.sort().map((t,i) => (
+              <option>{t}</option>
```

Comment:
Do you get a React warning for this? I would expect to need to assign [unique keys to each option](https://reactjs.org/docs/lists-and-keys.html#keys), like so:
```suggestion
              <option key={i}>{t}</option>
```

Reply:
I've updated this as part of the previous commit 😄